Source/WebInspectorUI/UserInterface/Models/ConsoleCommandResultMessage.js

```diff
@@ -29,7 +29,7 @@
 
 WI.ConsoleCommandResultMessage = class ConsoleCommandResult extends WI.ConsoleMessage
 {
-    constructor(target, result, wasThrown, savedResultIndex, shouldRevealConsole = true)
```

Comment:
a perfect example of why i dont like parameter defaults (other than `= { }`) 🥲

perhaps we should turn this into an `{ shouldRevealConsole } = { }` for `WI.ConsoleCommandResultMessage ` as well?